apps/designer/app/designer/features/style-panel/sections/spacing/layout.tsx

```diff
@@ -0,0 +1,302 @@
+import { styled, useId } from "@webstudio-is/design-system";
+import type { ComponentProps } from "react";
+import type { HoverTagret, SpacingStyleProperty } from "./types";
+import { spacingPropertiesNames } from "./types";
+
+const VALUE_WIDTH = 34;
```

Comment:
If someday we will decide to chaange width of `aside` we will get that internal container size is fixed?

<img width="359" alt="image" src="https://user-images.githubusercontent.com/5077042/208703306-20f48f3f-4cf5-4b4e-a454-5c29071d4745.png">

Mb calculations should start from aside theme value width? As looks like would be hard to find constants.
(IMHO any non-percentage width set inside the layout with known size would be an issue some day)

Reply:
I used base values as chosen in Figma:

<img width="363" alt="Screenshot 2022-12-20 at 19 45 56" src="https://user-images.githubusercontent.com/825702/208707483-f248e296-6e8e-4810-b56a-2be874031b12.png">